DayrSuryan/DSEthiop2.xml

```diff
@@ -252,16 +254,20 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            <seg type="rubrication">Incipits, chapter
                              titles, names Jesus and Christ, parts of the punctuation sign ፨,
                              days of the week written in the upper margin.</seg>
-                             <date notBefore="1850" notAfter="1949">Late 19th or first half of the 20th century.</date>
+
+                             <date>Late 19th or first half of the 20th century.</date>
+
                              <desc>Careful and regular, by a well-trained hand</desc>
                         </handNote>
 
                         <handNote xml:id="h2" script="Ethiopic" corresp="p1_i2">
                              <seg type="ink">Black, red.</seg>
                            <seg type="rubrication">Names of the months.</seg>
-                           <date notBefore="1850" notAfter="1949">Late 19th or first half of the 20th century.</date>
+
+                           <date>Late 19th or first half of the 20th century.</date>
                            <desc>Mediocre, letters of bigger size,
-                           very similar or possibly identical to <ref target="#h3">Hand 3</ref>.</desc>
+                           very similar or possibly identical <ref target="#h3"/>.</desc>
```

Comment:
I suggested in the comments to the PDF that we visualize the content of `<ref @target>`. I think that will be the easiest, instead of always coming up with different rules on how to call the different entities we refer to. How would you like `<ref target="#h3"/>` to be printed in the article?

Reply:
"identical to Hand 3", this is ok.